ostree-backup.sh

```diff
@@ -35,13 +44,26 @@ tar czf /var/tmp/backup/var.tgz \
     --exclude='/var/lib/cni/bin/*' \
     --selinux \
     /var
-ostree admin config-diff | awk '{print "/etc/" $2}' | xargs tar czf /var/tmp/backup/etc.tgz
+ostree admin config-diff | awk '{print "/etc/" $2}' | xargs tar czf /var/tmp/backup/etc.tgz --selinux
 rpm-ostree status > /var/tmp/backup/rpm-ostree.status
-ostree commit --branch backup /var/tmp/backup
+rpm-ostree status -v --json > /var/tmp/backup/rpm-ostree.json
+cp /etc/machine-config-daemon/currentconfig /var/tmp/backup/mco-currentconfig.json
+ostree commit --branch $backup_tag /var/tmp/backup
 
 # Create credentials for pushing the generated image
 mkdir /root/.docker
 cp backup-secret.json /root/.docker/config.json
 
 log_it Encapsulating and pushing backup OCI
-ostree container encapsulate backup registry:$backup_refspec --repo /ostree/repo --label ostree.bootable=true
+ostree container encapsulate $backup_tag registry:$backup_refspec --repo /ostree/repo --label ostree.bootable=true
+
+log_it Encapsulating and pushing base OCI
+base_commit=$(rpm-ostree status -v --json | jq -r '.deployments[] | select(.booted == true).checksum')
+ostree container encapsulate $base_commit registry:$base_refspec --repo /ostree/repo --label ostree.bootable=true
```

Comment:
I think what we should be doing here instead is just fetching the original upstream pristine container image, which we have in 
`rpm-ostree status --json | jq -r '.deployments[] | select(.booted == true)["container-image-digest"]'`